### PR TITLE
Updates for Convenience

### PR DESCRIPTION
**Pull request by gaboelnuevo**
* Add disabled prop
* Add images by default
* Tint Color prop for images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,8 +86,10 @@ For when you want to display an aggregate value. The rating will remain static u
 |**`spacing`**|number| pixel amount of separation between each star|No|`0`|
 |**`value`**|0 <= `value` <= `count`| (passing any value sets to Display Mode) star value to be displayed eg. 3.6 (of 5), 7.8 (of 10) |No|`null`|
 |**`backingColor`**|string/color| (in Display Mode) color behind the component (cannot be transparent)|No|`white`|
+|**`tintColor`**|string/color| Changes the color of all the non-transparent pixels to the tintColor |No|`null`|
 |**`opacity`**|bool|(in Display Mode) displays star values as opacity percentages (0 - 1.0)|No|`false`|
 |**`half`**|bool|(in Selection Mode) allows for half star ratings|No|`false`|
+|**`disabled`**|bool|Disable edition|No|`false`|
```
Comment (4umo) [inline README.md]: change the description to 'Disables onPress/buttons in Selection mode'

File: index.js [hunk 2]
```diff
@@ -145,11 +152,11 @@ export default class StarReview extends Component {
   star(val,starImg){
     return(
       <View key={val} style={{paddingLeft: this.props.spacing/2, paddingRight: this.props.spacing/2}}>
-        <TouchableOpacity onPress={()=>{
+        <TouchableOpacity disabled={this.props.disabled} onPress={()=>{
           this.setState({rating: val});
           this.props.update(val);
         }}>
-          <Image style={{width: this.props.starSize, height: this.props.starSize}} source={starImg} />
+          <Image style={[{width: this.props.starSize, height: this.props.starSize}, {tintColor: this.props.tintColor}]} source={starImg} />
```
Comment (4umo) [inline index.js]: Is there reason why this needs style array syntax? I'm not too familiar with its use but I'm wondering what it adds here.

**Review by 4umo — CHANGES_REQUESTED** [review]

**Review by gaboelnuevo — COMMENTED** [review]
Updated!

**Review by 4umo — APPROVED** [review]
looks good, thanks :)

**Pull request merged**